src/js/remoteServices.js

```diff
@@ -59,24 +59,17 @@ export default {
     allowHostOverride: false,
   },
 
-  "github-pr": {
+  "github": {
     name: "github",
     host: "https://github.com",
-    urlPatterns: [":host:/:org/:repo/pull/:id(/:tab)"],
-    id: (document, service, { org, repo, id }) => [service.key, org, repo, id].join("."),
-    description: (document) => document.querySelector(".js-issue-title")?.textContent?.trim(),
-    projectId: projectIdentifierBySelector(".js-issue-title"),
-    allowHostOverride: false,
-  },
-
-  "github-issue": {
-    name: "github",
-    host: "https://github.com",
-    urlPatterns: [":host:/:org/:repo/issues/:id"],
+    urlPatterns: [
+      ":host:/:org/:repo/issues/:id",
+      ":host:/:org/:repo/pull/:id(/:tab)"
+    ],
     id: (document, service, { org, repo, id }) => [service.key, org, repo, id].join("."),
-    description: (document, _service, { org: _org, repo: _repo, id: _id }) =>
-      document.querySelector(".js-issue-title")?.textContent?.trim(),
-    projectId: projectIdentifierBySelector(".js-issue-title"),
```

Comment:
Leave `projectId` here, other users might still rely on this.

Reply:
The `projectId` is still required here.

Reply:
@manubo Ahh sorry for not having this PR updated and applied your review.

I will try to finalize it in the next 2 weeks. Thx for bringing it up again!